go.mod

```diff
@@ -2,9 +2,15 @@ module github.com/dominant-strategies/quai-cpu-miner
 
 go 1.19
 
+replace github.com/dominant-strategies/go-quai => ../../djadih/go-quai
+
+replace github.com/dominant-strategies/go-quai-stratum => ../../djadih/go-quai-stratum
```

Comment:
We should not commit your `replace`ments. These are just for your local development/testing